elasticsearch/templates/statefulset.yaml

```
@@ -22,6 +22,7 @@ spec:
   podManagementPolicy: {{ .Values.podManagementPolicy }}
   updateStrategy:
     type: {{ .Values.updateStrategy }}
+  enableServiceLinks: false
```

```suggestion
  enableServiceLinks: {{ .Values.enableServiceLinks }}
```

with `enableServiceLinks` set to true by default in `values.yaml`